INSTALL.md

```diff
@@ -1,73 +1,144 @@
-Last Updated: March 28, 2017
+Last Updated: November 16, 2018
 
 ### Table of Contents
 
-**[Overview of Directories](#overview-of-directories)**
-
 **[Unpacking](#unpacking)**
 
-**[Setup](#setup)**
+**[Windows Setup](#windows-setup)**
+
+**[Non XAMPP Setup](#non-xampp-setup)**
+
+**[Disabling MySQL Strict Mode](#how-do-i-disable-mysql-strict-mode-?)**
 
 **[Setting up Access Control](#setting-up-access-control)**
 
 **[Upgrading](#upgrading)**
 
 **[FAQ](#faq)**
 
-**[Windows Setup](#windows-setup)**
+##  Unpacking
 
-##  Overview of Directories
+Download the source code directly from the repository located at [lh-ehr](https://github.com/LibreHealthIO/lh-ehr) using git bash.
 
-NOTE: Most recent documentation can be found on the online documentation at [LibreHealth](http://librehealth.io/).
+- First fork the repository.
 
-contrib: Contains many user-contributed encounter forms and utilities
-custom: Contains scripts and other text files commonly customized
-Documentation: Contains useful documentation
-interface: Contains User Interface scripts and configuration
-library: Contains scripts commonly included in other scripts
-sql: Contains initial database images
-gacl: Contains embedded php-GACL (access controls)
+- Then clone the fork.
 
-##  Unpacking
+ ```
+ git clone https://github.com/YOURUSERNAME/lh-ehr
+ ```
 
-The LibreHealthEHR release archive should be named as follows:
+Alternatively, you may download the release archive, which should be named as follows:
 
-`librehealthehr-<version>-release.tar.gz  -or-  librehealthehr-<version>-release.zip`
+`lh-ehr-<version>-release.tar.gz  -or-  lh-ehr-<version>-release.zip`
 
 To extract the archive, use either of the following commands from the command line:
 
-`bash# tar -pxvzf librehealthehr-<version>-release.tar.gz`
-`bash# unzip librehealthehr-<version>-release.tar.gz`
+`bash# tar -pxvzf lh-ehr-<version>-release.tar.gz`
+`bash# unzip lh-ehr-<version>-release.tar.gz`
 
 Be sure to use the `-p` flag when using tar, as certain permissions must be preserved.
 
-LibreHealthEHR will be extracted into a directory named `librehealthehr`.
+LibreHealthEHR will be extracted into a directory named `lh-ehr`.
 
-Alternatively you can download the source code directly from the repository located at [librehealthehr](https://github.com/LibreHealthIO/librehealthehr) using
+##  Windows Setup
 
-```
-git clone https://github.com/LibreHealthIO/LibreEHR librehealthehr
-```
+#### Requirements
+
+To run LibreHealthEHR on Windows, [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is needed. Make sure it is downloaded to 'C:\' directory to give XAMPP full permissions.
+
+**Note:**
+
+1. PHP [7.0](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/_) works and so does [5.6](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/) but [5.6.30](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is recommended
+
+2. PHP 7.1x is not currently supported
+
```

Comment:
You are recommending [unsupported versions of PHP](http://php.net/eol.php). At this point, PHP 7.1+ has to work.